apps/hash-api/src/index.ts

```diff
@@ -560,10 +560,10 @@ const main = async () => {
       }).pipe(
         Effect.provide(
           RpcClient.connectLayer(
-            Transport.multiaddr(`/ip4/${rpcHost}/tcp/${rpcPort}`),
+            Transport.multiaddr(`/dns/${rpcHost}/tcp/${rpcPort}`),
```

Comment:
thank you for going down the dns rabbit hole 😅 I kinda missed enabling that, but glad it was straightforward - out of curiosity - in typescript does a `/dns/127.0.0.1/tcp/4002` still work?

Reply:
Yes, I did not expected this actually but it just works. I was about to check if `rpcHost` is an IP or something else, but this is straight forward.

Reply:
Also, to be 100% sure I just tested it again locally